src/synchronization/stackImagePositionSynchronizer.js

```diff
@@ -15,7 +15,17 @@ export default function (synchronizer, sourceElement, targetElement) {
   const cornerstone = external.cornerstone;
   const sourceImage = cornerstone.getEnabledElement(sourceElement).image;
   const sourceImagePlane = cornerstone.metaData.get('imagePlaneModule', sourceImage.imageId);
-  const sourceImagePosition = convertToVector3(sourceImagePlane.imagePositionPatient);
+  let sourceImagePosition;
+
+  if (sourceImagePlane !== undefined) {
+    sourceImagePosition = convertToVector3(sourceImagePlane.imagePositionPatient);
+  }
+
+  if (sourceImagePosition === undefined) {
```

Comment:
I just realized there is no way this condition can happen. convertToVector3 doesn't have any way to return undefined. I guess it's better to move this If statement above the other one and check if sourceImagePlane.imagePositionPatient is undefined.

See https://github.com/chafey/cornerstoneTools/blob/55226c082b2e5ab2de57b6e2bdb82c072eafd516/src/util/convertToVector3.js#L9

Reply:
Done !